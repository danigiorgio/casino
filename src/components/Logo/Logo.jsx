import { Link } from 'react-router-dom';

const Logo = () => {
  return (
    <div className="ui one column center aligned page grid">
      <div className="column twelve wide">
        <Link to="/">
          <img src="/images/logo.svg" alt="logo" />
        </Link>
      </div>
    </div>
  );
};

export default Logo;
