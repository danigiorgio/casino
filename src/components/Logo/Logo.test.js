import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import Logo from './Logo';

describe('<Logo />', () => {
  it('should render Logo', () => {
    const { container } = render(
      <BrowserRouter>
        <Logo />
      </BrowserRouter>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
