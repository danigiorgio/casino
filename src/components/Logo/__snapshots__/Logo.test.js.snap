// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render Logo 1`] = `
<div
  class="ui one column center aligned page grid"
>
  <div
    class="column twelve wide"
  >
    <a
      href="/"
    >
      <img
        alt="logo"
        src="/images/logo.svg"
      />
    </a>
  </div>
</div>
`;
